draining_validation/hooks/pre-run.yml

```diff
@@ -8,13 +8,3 @@
     state: directory
     prefix: example-cnf-draining.
   register: cnf_tmp_dir
```

Comment:
Since we are not going to clone the example-cnf repo anymore, I think we can delete the creation of the temp folder, this also impacts to the post-run, and also probably we can do the same in launch_trex_job and upgrade_validation hooks.